[2021-02-28 09:16:35,270] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: my_first_dag.create_sample_file1 2021-02-28T08:16:29.182049+00:00 [queued]>
[2021-02-28 09:16:35,279] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: my_first_dag.create_sample_file1 2021-02-28T08:16:29.182049+00:00 [queued]>
[2021-02-28 09:16:35,279] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-02-28 09:16:35,279] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2021-02-28 09:16:35,279] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-02-28 09:16:35,288] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): create_sample_file1> on 2021-02-28T08:16:29.182049+00:00
[2021-02-28 09:16:35,290] {standard_task_runner.py:52} INFO - Started process 35185 to run task
[2021-02-28 09:16:35,297] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'my_first_dag', 'create_sample_file1', '2021-02-28T08:16:29.182049+00:00', '--job-id', '6', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/my_first_dag.py', '--cfg-path', '/var/folders/fh/z0rf64gj7d93005s7pxrkwm40000gn/T/tmp41qqijh7', '--error-file', '/var/folders/fh/z0rf64gj7d93005s7pxrkwm40000gn/T/tmpwhpw37fl']
[2021-02-28 09:16:35,299] {standard_task_runner.py:77} INFO - Job 6: Subtask create_sample_file1
[2021-02-28 09:16:35,327] {logging_mixin.py:104} INFO - Running <TaskInstance: my_first_dag.create_sample_file1 2021-02-28T08:16:29.182049+00:00 [running]> on host 120.0.168.192.in-addr.arpa
[2021-02-28 09:16:35,349] {taskinstance.py:1255} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=my_first_dag
AIRFLOW_CTX_TASK_ID=create_sample_file1
AIRFLOW_CTX_EXECUTION_DATE=2021-02-28T08:16:29.182049+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-02-28T08:16:29.182049+00:00
[2021-02-28 09:16:35,350] {bash.py:135} INFO - Tmp dir root location: 
 /var/folders/fh/z0rf64gj7d93005s7pxrkwm40000gn/T
[2021-02-28 09:16:35,351] {bash.py:158} INFO - Running command: 
            cd /Users/yuyatinnefeld/Desktop/projects/py_airflow/data
            touch sample1.txt
            date >> sample1.txt
[2021-02-28 09:16:35,357] {bash.py:169} INFO - Output:
[2021-02-28 09:16:35,363] {bash.py:177} INFO - Command exited with return code 0
[2021-02-28 09:16:35,376] {taskinstance.py:1159} INFO - Marking task as SUCCESS. dag_id=my_first_dag, task_id=create_sample_file1, execution_date=20210228T081629, start_date=20210228T081635, end_date=20210228T081635
[2021-02-28 09:16:35,388] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-02-28 09:16:35,433] {local_task_job.py:146} INFO - Task exited with return code 0
